packages/adp-tooling/src/preview/routes-handler.ts

```diff
@@ -112,4 +117,104 @@ export default class RoutesHandler {
             next(e);
         }
     };
+
+    /**
+     * Handler for reading all controller js files from the workspace.
+     *
+     * @param _ Request
+     * @param res Response
+     * @param next Next Function
+     */
+    public handleReadAllControllers = async (_: Request, res: Response, next: NextFunction) => {
+        try {
+            const files = await this.project.byGlob('/**/changes/coding/*.js');
```

Comment:
I don't think you should limit this to `coding`, `this.project.byGlob('/**/changes/**/*.controller.js');` is better because technically I can put my code anywhere under changes even if the tooling uses this as default (right now)

Reply:
Changed in: https://github.com/SAP/open-ux-tools/pull/1275/commits/dbded1d478b30424a53ef16dc6720a5fd654354a

Reply:
This might be valid for the preview, but when the project is deployed, there is a strict check for the structure, thus:
- not all JS files should match, because only the files under `changes/coding` are controller extensions.
- the same is valid for the fragments - only fragments in `changes/fragments` should match the pattern.

Reply:
Oops. I didn't know that. Then we should for sure also enforce the rules here.
@nikmace sorry for pushing you in the wrong direction.